osbs/build/user_params.py

```diff
@@ -97,18 +99,92 @@ def value(self, val):  # pylint: disable=W0221
         BuildParam.value.fset(self, registry_uris)  # pylint: disable=no-member
 
 
+def load_user_params_from_json(user_params_json):
+    """Load user params from json into proper object
+
+    :param str user_params_json: json with user params
+    :rtype: subclass of BuildCommon
+    :return: initialized object with user params
+    """
+    BUILD_KINDS_MAP = {
+        BuildUserParams.KIND: BuildUserParams,
```

Comment:
I may create register decorator instead, WDYT?

Reply:
I think a decorator would be nicer, yes.

Reply:
done